### DISCUSSION TITLE
Modify Lotus Node Chain Storage Location

### DISCUSSION THREAD

**Discussion by LoganC034** [filecoin-project/lotus#9235] [General Q&A] [answered]
I am having trouble finding clear documentation as to how one changes the storage location from the default ~./lotus. For that matter, is there a way to change the installation directory from the users home also? Where would one modify the configuration in general? I've seen some things reference ~/.lotus/config. Which I do not have. Do I need to create one? Even still, looking at that documentation I do not see file paths. I  was hoping to do all of this before launching a daemon.

**Comment by stuberman** [comment, marked as answer]
There are several env path variables you can set for .lotus and .lotusminer to change the location.  

Did you search lotus.FileCoin.io?

**Reply by LoganC034** [reply]
Thanks for responding. I did search the site, and did not find anything about seting env variables. Perhaps i missed it?

**Reply by LoganC034** [reply]
I spoke too soon. Found it here - https://lotus.filecoin.io/lotus/configure/defaults/ I was focused on the config file and did not scroll far enough down.
